app/grandchallenge/components/forms.py

```diff
@@ -118,7 +113,7 @@ def __init__(self, *args, instance, base_obj, user, **kwargs):
 
         # add fields for all interfaces that already exist on
         # other display sets / archive items
-        for slug, values in base_obj.values_for_interfaces.items():
+        for slug, _ in base_obj.values_for_interfaces.items():
```

Comment:
Tip: if you're just discarding the values you can drop the `.items()` to just iterate over the keys.